packages/jsii-pacmak/lib/generator.ts

```diff
@@ -110,19 +142,8 @@ export abstract class Generator implements IGenerator {
   /**
    * Returns the file name of the assembly resource as it is going to be saved.
    */
-  protected getAssemblyFileName() {
-    let name = this.assembly.name;
-    const parts = name.split('/');
-
-    if (parts.length === 1) {
-      name = parts[0];
-    } else if (parts.length === 2 && parts[0].startsWith('@')) {
-      name = parts[1];
-    } else {
-      throw new Error('Malformed assembly name. Expecting either <name> or @<scope>/<name>');
-    }
-
-    return `${name}@${this.assembly.version}.jsii.tgz`
+  protected getAssemblyFileName(): string {
+    return this.getAssemblyArchiveFileName(this.reflectAssembly)
```

Comment:
Missing `;` -- I somehow thought I had an `eslint` rule configured to catch this... If that's not the case I have to re-verify that... 😅